spec/compiler/semantic/named_tuple_spec.cr

```diff
@@ -48,6 +62,28 @@ describe "Semantic: named tuples" do
       )) { nil_type }
   end
 
+
+  it "types nilable named tuple access (4)" do
+    assert_type(%(
+      t = {x: 1, y: 'a'}
+      t["x"]?
+      )) { int32 }
+  end
+
+  it "types nilable named tuple access (5)" do
+    assert_type(%(
+      t = {x: 1, y: 'a'}
+      t["y"]?
+      )) { char }
+  end
+
+  it "types nilable named tuple access (6)" do
+    assert_type(%(
+      t = {x: 1, y: 'a'}
+      t["foo"]?
+      )) { nil_type }
+  end
+
```

Comment:
Not sure if that's the right place for these specs, but couldn't find anything else.